canonical-web-and-design/charmhub.io
# Remove cs prefix. juju will now deploy only with charm name

## Done
- From now on juju will deploy with the name that is saved in charmhub store.
- We are removing the exceptions that we were making for some charms.

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
 - Try a few charms details pages
 - make sure that the command is only: `juju deploy {{charmname}}`


## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1867
